specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-11-01-preview/examples/CreateClusterPoolWithUDRAks.json

```diff
@@ -0,0 +1,107 @@
+{
+  "parameters": {
+    "subscriptionId": "10e32bab-26da-4cc4-a441-52b318f824e6",
+    "resourceGroupName": "hiloResourcegroup",
+    "api-version": "2023-11-01-preview",
+    "clusterPoolName": "clusterpool1",
+    "ClusterPool": {
+      "location": "West US 2",
+      "properties": {
+        "clusterPoolProfile": {
+          "clusterPoolVersion": "1.2"
+        },
+        "computeProfile": {
+          "vmSize": "Standard_D3_v2"
+        },
+        "networkProfile": {
+          "subnetId": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
+          "outboundType": "userDefinedRouting"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.HDInsight/clusterPools/clusterpool1",
+        "name": "clusterpool1",
+        "type": "Microsoft.HDInsight/clusterPools",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "deploymentId": "45cd32aead6e4a91b079a0cdbfac8c36",
+          "managedResourceGroupName": "hdi-45cd32aead6e4a91b079a0cdbfac8c36",
+          "clusterPoolProfile": {
+            "clusterPoolVersion": "1.2"
+          },
+          "computeProfile": {
+            "vmSize": "Standard_D3_v2",
+            "count": 3
+          },
+          "aksClusterProfile": {
+            "aksVersion": "1.24",
+            "aksClusterAgentPoolIdentityProfile": {
+              "msiResourceId": "/subscriptions/subid/resourceGroups/hdi-45cd32aead6e4a91b079a0cdbfac8c36/providers/Microsoft.ManagedIdentity/userAssignedIdentities/clusterpool1-agentpool",
+              "msiClientId": "a89fb478-2a84-4d9b-8f18-3e8c4d1db3eb",
+              "msiObjectId": "dc7ef861-8b55-4ffb-9003-20885cd895a9"
+            },
+            "aksClusterResourceId": "/subscriptions/subid/resourceGroups/hdi-45cd32aead6e4a91b079a0cdbfac8c36/providers/Microsoft.ContainerService/managedClusters/clusterpool1"
+          },
+          "networkProfile": {
+            "subnetId": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
+            "outboundType": "userDefinedRouting"
```

Comment:
>   "outboundType": "userDefinedRouting" [](http://example.com/codeflow?start=9&length=39)

outboundType seems like a slightly funny name to me. How about "outboundRouting"?